test/HighlightTest.php

```diff
@@ -39,4 +40,70 @@ public function testUnknownLanguageThrowsDomainException()
         $hl = new Highlighter();
         $hl->highlight("blah++", "als blurp eq z dan zeg 'flipper'");
     }
+
+    public function testListLanguagesWithoutAliases()
+    {
+        $expectedLanguageCount = count(glob(__DIR__ . DIRECTORY_SEPARATOR . '..' . DIRECTORY_SEPARATOR .
```

Comment:
For our unit tests, we have [Symfony's Finder component](https://symfony.com/doc/current/components/finder.html) for convenience, so you could make use of it. See our [`MarkupTest.php`](https://github.com/scrivo/highlight.php/blob/224929083dbec2d1e3f7adda120a0ed5c9485d38/test/MarkupTest.php#L40-L46)

```php
$finder = new Finder();
$expectedLanguageCount = $finder
    ->in(__DIR__ . '/../Highlight/languages/')
    ->name('*.json')
    ->count()
;
```

Reply:
Is there anything convenient that would work well within the `testGetAliasesForLanguageWhenUsingMainLanguageName` as well?